server/src/test/java/com/objectcomputing/checkins/security/GoogleServiceConfigurationTest.java

```diff
@@ -30,14 +30,16 @@ void testInstantiation() {
         assertNull(googleServiceConfiguration.getToken_uri());
```

Comment:
A lot of tests are failing for me with `Message: Could not resolve placeholder ${DIRECTORY_ID}`. I think you have this set up locally but need to set it up in the test environment.

Reply:
We've fixed this now. We do not need service account details for testing purpose. We've mocked all the calls that need this.